app/styles/design-system/components/link.css

```
@@ -0,0 +1,69 @@
+@layer reset {
+    a,
+    .link {
+        text-decoration-color: currentcolor;
+        text-decoration-thickness: 0.125em;
+        text-underline-offset: 0.2em;
+        text-decoration-skip-ink: auto;
+        text-decoration: underline;
```

Stylelint (17.6.0)

[error] Unexpected shorthand "text-decoration" after "text-decoration-color" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)